test/rbs/test/runtime_test_test.rb

```diff
@@ -68,10 +68,10 @@ def move(x: 0, y: 0)
           "RBS_TEST_OPT" => "-I./foo.rbs"
         }
         ruby = ENV['RUBY'] || RbConfig.ruby
-        command_line = if defined?(Bundler)
-                         [ruby, "-rbundler/setup", "-rrbs/test/setup", "sample.rb"]
-                       else
+        command_line = if ENV['RUBY']
```

Comment:
Seems like we can omit the `ruby` var, but it's up to you. (If you have another fix...)

Reply:
Thanks, I removed the `ruby` var. :put_litter_in_its_place: